sup3r/utilities/topo.py

```diff
@@ -242,8 +242,38 @@ class will output a topography raster corresponding to the
             topo_source_h5, usually meters.
         """
 
-        te = cls(file_paths, topo_source_h5, s_enhance, agg_factor,
+        te = cls(file_paths, topo_source, s_enhance, agg_factor,
                  target=target, shape=shape, raster_file=raster_file,
                  max_delta=max_delta, input_handler=input_handler)
 
         return te.hr_elev
+
+
+class TopoExtractNC(TopoExtract):
```

Comment:
consider making an abstract base class with two sub classes (one for h5 and one for nc) with the two property methods over-written. Better organization and explicit subclass purposes that way.

Reply:
Yeah agreed